blockchain/genesis/genesis.go

```diff
@@ -196,6 +196,11 @@ func New() (Genesis, error) {
 	return genesis, nil
 }
 
+// PacificHeight returns the pacific height
+func PacificHeight() uint64 {
```

Comment:
We shouldn't always assume the default config. Test and testnet are using different ones.